addon/components/resize-sensor.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+import layout from '../templates/components/resize-sensor';
+
+import ResizeSensor from 'resize-sensor';
+
+export default Ember.Component.extend({
+  layout,
+  classNames: ['resize-sensor-container'],
+  didInsertElement() {
+    const yieldHtml = this.$().children()[0];
+    this._resizeSensor = new ResizeSensor(yieldHtml, () => {
```

Comment:
Generally speaking we want to avoid using Jquery - in the long term, Jquery is going to be removed from Ember directly as a dependency. You can access the element for this component directly:

```js
this._resizeSensor = new ResizeSensor(this.element, () => {});
```